front/src/Components/ActionBar/ActionBar.svelte

```diff
@@ -446,6 +452,17 @@
                         <img draggable="false" src={menuImg} style="padding: 2px" alt={$LL.menu.icon.open.menu()} />
                     </button>
                 </div>
+                {#if gameManager.getCurrentGameScene().isMapEditorEnabled()}
```

Comment:
TODO: check if this works when switching from a non editable to an editable map (we need to make sure the ActionBar component is destroyed and recreated when we change map)